acceptance/acceptance_test.go

```diff
@@ -901,6 +867,8 @@ func testAcceptance(
 						var buildpackTgz, tempVolume string
 
 						it.Before(func() {
+							h.SkipIf(t, os.Getenv("DOCKER_HOST") != "", "cannot mount volume when DOCKER_HOST is set")
```

Comment:
Not currently, but since it's a docker usage concern, I wonder if we should link to the [docker docs](https://docs.docker.com/engine/reference/commandline/cli/) somewhere?

Reply:
Maybe this could be covered in https://github.com/buildpacks/docs/issues/180?

Reply:
You might add "pack developer" to the "target audience" list in that issue.

Reply:
Is that audience covered by buildpacks.io??

Reply:
It's not an I don't think it should. "pack developers" should be looking at the repo *.md or wiki.